packages/vue-components/tsconfig.app.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "skipLibCheck": true,
+
+    /* Bundler mode */
```

Comment:
Those comments in JSON files are not problematic?

Reply:
nope, they are not ;-) in fact, those files are [json5](https://json5.org/)